ajax/libs/kiss.animate/package.json

```diff
@@ -12,5 +12,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/daogurtsov/KISS.Animate.git"
-  }
+  }, 
```

Comment:
@kennynoah Should I make a new request?
Or how can I modify the content directly?